package.json

```diff
@@ -84,7 +84,7 @@
   },
   "dependencies": {
     "@hapi/joi": "15.1.1",
-    "@hapi/joi16": "npm:@hapi/joi@16.1.1",
+    "@hapi/joiNext": "npm:@hapi/joi@17.1.1",
```

Comment:
Renamed this `joiNext` as having `joi16` as an alias for joi 17 gave me a headache.